brigade-controller/cmd/brigade-controller/main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 	flag.StringVar(&ctrConfig.WorkerRequestsMemory, "worker-requests-memory", "", "kubernetes worker memory requests")
 	flag.StringVar(&ctrConfig.WorkerLimitsCPU, "worker-limits-cpu", "", "kubernetes worker cpu limits")
 	flag.StringVar(&ctrConfig.WorkerLimitsMemory, "worker-limits-memory", "", "kubernetes worker memory limits")
+	flag.BoolVar(&ctrConfig.AllowSecretKeyRef, "allow-secret-key-ref", true, "allow jobs to use secretKeyRef in their environment")
```

Comment:
Again, I think we want this to be a per-project setting, not a global one. What do you think?

Reply:
Similar to the service account we could have a global default with the ability for projects to override?

Reply:
Yes, I think that would be the right approach.
Thanks!

Reply:
Actually while adding this it probably isn't needed since golang bools will default to false so projects will always have it turned off unless set to true.